src/sagetasks/utils.py

```diff
@@ -12,3 +14,23 @@ def to_prefect_tasks(module_name, general_module):
         first_line = docstring.splitlines()[0]
         task_func = task(func, name=first_line)
         setattr(this_module, name, task_func)
+
+
+def update_dict(base_dict, overrides):
```

Comment:
Nit: I'm alright with these not having docstrings as the function basically says what it does.